packages/oracle/data_stream/memory/_dev/test/system/test-memory-config.yml

```
@@ -8,16 +8,34 @@ agent:
     contents: |
       set -eu
       if grep wolfi /etc/os-release > /dev/null ; then
-        apk update && apk add libaio wget unzip
+        apk update
+        apk add libaio wget unzip
+      elif grep "Red Hat" /etc/os-release > /dev/null ; then
```

Nit. `-q` can be used to avoid printing the matching lines.
```suggestion
      elif grep -q "Red Hat" /etc/os-release ; then
```

---

Thanks!
Parameter `-q` has been added for all config files in https://github.com/elastic/integrations/pull/12627/commits/fd9d5c88811f8d620b0f1f2eee4f95323208a2ff